scipy/sparse/sparsetools/csr.h

```diff
@@ -776,7 +801,7 @@ void csr_binop_csr_general(const I n_row, const I n_col,
  * Note:
  *   Input:  A and B column indices are assumed to be in sorted order
  *   Output: C column indices will be in sorted order
- *           Cx will not contain any zero entries
+ *           Cx will not contain any implicit zero entries
```

Comment:
I think you meant "explicit" here.

Reply:
`Cx` initially contained only the non-zero values but after these changes it can also have the explicit zero values. That means `Cx` will have the explicit zeros but not the implicit ones.